siibra/NeuroglancerAdapter.m

```diff
@@ -0,0 +1,71 @@
+%% STEP 1: INHERIT FROM ADAPTER CLASS
+classdef NeuroglancerAdapter < images.blocked.Adapter
+    
+    properties
+        uri (1,1) string
+        Info (1,1) struct
+        level_key string
+    end    
+    
+%% STEP 2: DEFINE REQUIRED METHODS
+    methods
+        
+        % Define the openToRead method
+        function openToRead(obj,source)
+            obj.uri = source;
+        end
+
+        % Define the getInfo method      
+        function info = getInfo(obj)
+            
+            % Read  info
+            import matlab.net.*
```

Comment:
Can you identify one or more key requirements that drove the use of the low-level `matlab.net.*` interface vs the higher-level `webread` interface?

Reply:
I needed a way to specify the content type. Thank you for pointing me to `webread` this is much more concise and it allows setting the content type as well.